Graphs/DynamicBarGraph/barGraph.test.js

```diff
@@ -0,0 +1,550 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { checkTicks, getDataAndConfig, getHtml, checkSvg } from '../testHelper';
+import BarGraph from '.';
+
+const cheerio = require('cheerio');
+
+describe("Bar Graph", () => {
+    let config;
+    beforeAll(async () => {
+        config = await getDataAndConfig('DynamicBarGraph');
+    });
+
+    describe("Vertical Bar", () => {
+        describe("OtherOption by Number", () => {
+            let verticalNumber, $;
+            beforeAll(async (done) => {
+                verticalNumber = await mount(
+                    <BarGraph
+                        width={500}
+                        height={500}
+                        configuration={config.verticalNumber}
+                        data={config.data}>
+                    </BarGraph>
+                );
+                /* Delayed added because bar is rendered with 300ms animation */
+                setTimeout(() => {
+                    verticalNumber.update();
+                    done();
+                }, 350);
+            });
+
+            it("SVG Dimensions", () => {
+                const result = checkSvg(verticalNumber);
+                expect(result).toBeTruthy();
+            });
+
+            it("Total Bars", () => {
+                $ = getHtml(verticalNumber, '.graph-bars');
+                const noOfBars = $('.graph-bars').find('g').length;
+                expect(noOfBars).toBe(6);
+            });
+
+            it("xAxis Ticks Length", () => {
+                const xAxisTicks = checkTicks(verticalNumber, '.graph-container', '.xAxis', 'g')
+                expect(xAxisTicks).toBe(6);
+            });
+
+            it("yAxis Ticks Length", () => {
+                const yAxisTicks = checkTicks(verticalNumber, '.graph-container', '.yAxis', 'g')
+                expect(yAxisTicks).toBe(6);
+            });
+
+            it("Bar Dimensions Height, Width and Positions", () => {
+                $ = getHtml(verticalNumber, '.graph-bars');
+                const bar = $('.graph-bars').children().first().next().find('rect');
```

Comment:
Could we do it for at least two bars? and create a helper function to return the dimension of the bar as an object and then directly compare that object with lodash? and all other areas. That will reduce down the coding.